var/spack/repos/builtin/packages/rnpletal/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Rnpletal(Package):
```

Comment:
We should probably use `AutotoolsPackage` as the base class. You can override `configure`, `build`, and `install` to iterate over each package. Although you only seem to only build one package anyway, so you can just set `configure_directory = "rnpl"`

Reply:
I foresee having to build other packages as well in the future, in particular `visutil`. I converted it to an `AutotoolsPackage`, but I kept the explicit loops over the packages.